src/SfxWeb/src/app/Common/Constants.ts

```diff
@@ -152,7 +152,9 @@ export class SortPriorities {
         None: 1,
         Primary: 2,
         ActiveSecondary: 3,
-        IdleSecondary: 4
+        IdleSecondary: 4,
+        ActiveSecondaryAuxiliary: 5,
```

Comment:
Should we have this naming consistent with SF code ? ActiveAuxiliary and IdleAuxiliary. Also don't we need PrimaryAuxiliary here?

Reply:
I believe you're looking at an outdated version of the file, you can look at files changed tab to get latest view

Reply:
Oh yeah .. You are right about it